src/render_iframe.js

```diff
@@ -304,8 +304,8 @@ EPUBJS.Render.Iframe.prototype.scroll = function(boolX, boolY, type){
         }
 	} else {
         if (this.isMobile) {
-            parent.style.overflowX = "";
-            parent.style.overflowY = "";
+            parent.style.overflowX = "hidden";
+            parent.style.overflowY = "hidden";
             parent.style[this.overflowScrolling] = "";
```

Comment:
For mobile device, the iframe is already scrollable, if we do not disable parent's scrolling it may show two scrollbars that is just weird and may not feel right for touch input.